pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/BaseMigrator.java

```diff
@@ -90,7 +94,16 @@ private Map<String, Object> replaceLegacyAltPidKeysWithGuids(Map<String, Object>
     @Override
     public void export() {
         Map<String, Object> dataByRealm = getDataByRealm();
+        if (index != null) {
+            participantsInTheStudy = elasticSearch.getAllParticipantsInIndex(index);
```

Comment:
I think `participantsInTheStudy` should be have method scope--otherwise it'll stick around and take up space when it's not needed after the export.